frontend/components/analytics/GoogleAnalytics.tsx

```diff
@@ -1,35 +1,54 @@
 import { GA_URL, GA_KEY, GA_DEBUG } from '@config/env';
-import Head from 'next/head';
+import * as React from 'react';
+import Script from 'next/script';
+import { useRouter } from 'next/router';
+
+declare const ga: (command: string, hitType: string, url?: string) => void;
 
 export function GoogleAnalytics() {
+   const router = useRouter();
+   React.useEffect(() => {
+      const handleRouteChange = (url: string) => {
```

Comment:
Do we need to do this same thing for Matomo?

Reply:
I'll check if navigation is working, didn't think of that.

Reply:
It doesn't seem like Matomo knows about route changes. I made an issue: https://github.com/iFixit/react-commerce/issues/595